commands/deployments/tasks/dep_update_task_step_state.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tasks
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"path"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ystia/yorc/v4/commands/deployments"
+	"github.com/ystia/yorc/v4/commands/httputil"
+	"github.com/ystia/yorc/v4/tasks"
+)
+
+func init() {
+	tasksCmd.AddCommand(updateTaskStepStateCmd)
+}
+
+func updateTaskStepState(deploymentID, taskID, stepName, statusStr string) {
+	client, err := httputil.GetClient(deployments.ClientConfig)
+	if err != nil {
+		httputil.ErrExit(err)
+	}
+
+	statusStr = strings.ToLower(statusStr)
+
+	// Safety check
+	_, err = tasks.ParseTaskStepStatus(statusStr)
+	if err != nil {
+		httputil.ErrExit(err)
+	}
+
+	// The task step status is set to "done"
```

Comment:
Maybe remove the comment: the task step status is set to done or initial depending on the argument value

Reply:
Right! Copy/Paste error let remove it